cmd/podman/rmi.go

```diff
@@ -91,8 +91,23 @@ func rmiCmd(c *cli.Context) error {
 		if err != nil {
 			return errors.Wrapf(err, "unable to query local images")
 		}
-		for _, i := range imagesToDelete {
-			removeImage(i)
+		lastNumberofImages := 0
+		for len(imagesToDelete) > 0 {
```

Comment:
Do we want to do this without a check that --force is in effect?  I think the current behavior with just --all is correct.  But if the user wants to wipe all by selecting the --force, then buyer beware.

Reply:
force just deletes the containers based on the image itself.  And that logic is in the gofunc preceeding this code.

Reply:
With the fix, it's removing the images without at least a warning as I suspected it would without the --force flag and I think it should not.

```
## Installed Version ##
# /usr/bin/podman -v
podman version 0.10.1.3

# cat ~/Dockerfile.multirun
FROM docker.io/library/alpine:latest AS base
RUN touch /1
ENV LOCAL=/1
RUN find $LOCAL
FROM base
RUN find $LOCAL

# /usr/bin/podman build -t tom -f ~/Dockerfile.multirun . 
STEP 1: FROM docker.io/library/alpine:latest AS base
Getting image source signatures
{removed happy build output for brevity}
STEP 10: COMMIT tom

# /usr/bin/podman rmi -a
Untagged: docker.io/library/alpine:latest
unable to delete "880a44052fc2bbebaaa978c2de37d2c61de773e177a05a5f4c29a3fedb25275f" (cannot be forced) - image has dependent child images
unable to delete "368a1af94b94fa2c319cef822c3d0ca6ee62c2132781f0364c57ca6d19c1ca1a" (cannot be forced) - image has dependent child images
d419b1ffeb6fde3a515c8ac36c1196f82a65fd0f880158cf7ebe633745325fc3
368a1af94b94fa2c319cef822c3d0ca6ee62c2132781f0364c57ca6d19c1ca1a
880a44052fc2bbebaaa978c2de37d2c61de773e177a05a5f4c29a3fedb25275f
196d12cf6ab19273823e700516e98eb1910b03b17840f9d5509f03858484d321
7674ea514496bdc55083ace98e66ea6452bf6133229e0f4b71db5139bcc7a352
unable to delete "d419b1ffeb6fde3a515c8ac36c1196f82a65fd0f880158cf7ebe633745325fc3" (cannot be forced) - image has dependent child images

# buildah rmi -a

## New Dev Version with Fix ##

# podman -v
podman version 0.11.2-dev

# podman build -t tom -f ~/Dockerfile.multirun .
STEP 1: FROM docker.io/library/alpine:latest AS base
Getting image source signatures
{Extra happy build info removed for brevity}
STEP 10: COMMIT tom

# podman rmi -a
cd8d85c2262031bf526ea11e117ac7cf2f18fdaa23915032b2a78909856e1991
825dbb85ab87965644e9490a7d925cdb9b142ca9e60163b08639c8b41ec260b2
e77a5284fdab744add61237631c0c78679dee865afff64d2bd23b53f8b8dd411
9246e830622ac8efaabfe84db1b6488ce65e649f683482748d040b44cdb2b695
196d12cf6ab19273823e700516e98eb1910b03b17840f9d5509f03858484d321

```